build.gradle

```diff
@@ -59,11 +59,7 @@ tasks.register('nightly') {
 tasks.register('prepareCompose') {
     it.group 'Deephaven lifecycle'
     it.description 'A lifecycle task that prepares prerequisites for local docker-compose builds'
-    it.dependsOn project(':docker-server-slim').tasks.findByName('buildDocker-server-slim'),
-            project(':docker-server').tasks.findByName('buildDocker-server'),
-            project(':docker-web').tasks.findByName('buildDocker'),
-            project(':envoy').tasks.findByName('buildDocker'),
```

Comment:
to confirm, CI will still build and publish these, right?

Reply:
Correct.